### PR TITLE
Some changes for cube data in vasputil

### PR DESCRIPTION
**Pull request by sigsid**
- Remove division by 10 of radius
- Add radii to dataframe
- Fix error in parser

### CODE REVIEW

File: misc/vasp/python/vasputil.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def parseFile(file):
         pos.append(list(map(float, line.strip().split())))
     pos = numpy.array(pos).astype(numpy.float32)
 
-    factor = scale if scale >= 0.0 else np.power(
+    factor = scale if scale >= 0.0 else numpy.power(
         -scale / np.dot(a3, np.cross(a2, a1)), 1/3)
```
Comment (tbmasood) [inline misc/vasp/python/vasputil.py:45]: Please change "np" to "numpy" in this line as well.

**Review by tbmasood — CHANGES_REQUESTED** [review]
Looks good. Just one more minor change required.

File: misc/vasp/python/vasputil.py (hunk 1 above)
Comment (sigsid) [reply misc/vasp/python/vasputil.py]: Yes of course, I have changed it now

**Review by tbmasood — APPROVED** [review]
Looks good now 👍

**Pull request merged**